west-sbom-report.md

```diff
@@ -0,0 +1,104 @@
+> ### WARNING!
+> Some of the files does not contains license information or the license contained in them cannot be detected.<br>
+See [license details](internal_link) for list of files.
+Evaluate them manually to get license information.
+
+
+# Summary
+
+This report was generated from the following inputs:
+* `zephyr/zephyr.elf from build directory /home/doki/work/ncs2/nrf/samples/crypto/ecdsa/build_sbom_test`
+* `files: **/*.c, **/*.h (relative to /home/doki/work/ncs2/nrf/samples/crypto/ecdsa/)`
+* `list of files read from: /home/doki/work/ncs2/sample-list.txt`
+
+Detected files are covered by the licenses:
+* [LicenseRef-Nordic-5-Clause OR Apache-2.0](internal_link)
+* [LicenseRef-Nordic-5-Clause](internal_link)
+* [Apache-2.0](internal_link)
+* Possibly other licenses that [cannot be detected automatically](internal_link)
+
+
+Following licenses are not covered by [SPDX License List](internet_link), so texts are added to this report:
+* [LicenseRef-Nordic-5-Clause](internal_link)
+
+# License details
+
+## No license information detected
+```
+/home/doki/work/ncs2/nrf/samples/crypto/ecdsa/build_sbom_test/zephyr/dev_handles.c
+/home/doki/work/ncs2/nrf/samples/crypto/ecdsa/build_sbom_test/zephyr/include/generated/autoconf.h
+/home/doki/work/ncs2/nrf/samples/crypto/ecdsa/build_sbom_test/zephyr/include/generated/device_extern.h
+/home/doki/work/ncs2/nrf/samples/crypto/ecdsa/build_sbom_test/zephyr/include/generated/devicetree_fixups.h
+```
+
+## LicenseRef-Nordic-5-Clause OR Apache-2.0
+
+License expression with following licenses:
+
+ * Non-standard SPDX License: Nordic 5-Clause License
+
+   See [license text below](internal_link).
+ 
+ * Apache License 2.0
+
+   https://spdx.org/licenses/Apache-2.0.html
+
+```
+/home/doki/work/ncs2/nrf/drivers/entropy/entropy_cc3xx.c
+/home/doki/work/ncs2/nrf/drivers/hw_cc310/hw_cc310.c
+/home/doki/work/ncs2/nrf/lib/fatal_error/fatal_error.c
+```
+
+## Apache-2.0 OR WITH AND OR AND
+
+License expression with that cannot be correctly interpreted
+
+```
+/home/doki/work/ncs2/nrf/drivers/entropy/entropy_cc3xx.c
+/home/doki/work/ncs2/nrf/drivers/hw_cc310/hw_cc310.c
+/home/doki/work/ncs2/nrf/lib/fatal_error/fatal_error.c
+```
+
+## LicenseRef-Nordic-5-Clause
+
+Non-standard SPDX License: Nordic 5-Clause License
+
+See [license text below](internal_link).
+
+```
+/home/doki/work/ncs2/nrf/drivers/entropy/entropy_cc3xx.c
+/home/doki/work/ncs2/nrf/drivers/hw_cc310/hw_cc310.c
+/home/doki/work/ncs2/nrf/lib/fatal_error/fatal_error.c
+```
+
+## Apache-2.0
+
+Apache License 2.0
+
+https://spdx.org/licenses/Apache-2.0.html
```

Comment:
```suggestion
[Apache License 2.0](https://spdx.org/licenses/Apache-2.0.html)
```

Reply:
In this and other paces, I wanted to separate URL from license name to:
* show user that the details are outside this report
* show origin of the license (spdx.org in this case)

Reply:
What if you have them as suggested and add some text, like "outside this report" or "not included in this report", after the link for reports like that?